oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -175,6 +180,9 @@ private ConditionalMapPropertySource(String name, Map<String, Object> source, En
 
         @Override
         public Object getProperty(String name) {
+            if (OKTA_ISSUER_URI.equals(name) && OKTA_STATIC_DISCOVERY.equals(this.getName()) && !containsProperty(name)) {
```

Comment:
Ideally, this validation logic should be live here: 

https://github.com/okta/okta-spring-boot/blob/8b8a58ca5048abcc9f3b372583a4af4ac628b018/oauth2/src/main/java/com/okta/spring/boot/oauth/config/OktaOAuth2Properties.java#L163

Reply:
The application will crash much earlier, long before this validation.
That's what I'm trying to say: "approach with validators does't work for this property"